shell/common/native_mate_converters/value_converter.cc

```diff
@@ -39,7 +40,7 @@ bool Converter<base::Value>::FromV8(v8::Isolate* isolate,
   std::unique_ptr<base::Value> value(
       converter.FromV8Value(val, isolate->GetCurrentContext()));
```

Comment:
Can we avoid creating a unique_ptr here just for temporary storage ?

Reply:
The `V8ValueConverter::FromV8Value` returns `std::unique_ptr` as result so it can not be avoided.

I tried to refactor `V8ValueConverter` to make it return `base::Value` directly, but it is then impossible to distinguish between conversion failures and null results.